src/MaterialDesignThemes.Wpf/Converters/TextFieldClearButtonVisibilityConverter.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace MaterialDesignThemes.Wpf.Converters;
 
-internal class TextFieldClearButtonVisibilityConverter : IMultiValueConverter
+public class TextFieldClearButtonVisibilityConverter : IMultiValueConverter
```

Comment:
I propose moving this inside of a new `MaterialDesignThemes.Wpf.Converters.Internal` namespace.